util/commands/audio.schema.json

```diff
@@ -0,0 +1,230 @@
+{
+    "$id": "https://kulturkrock.se/audiocommand.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Audio commands in Opus",
+    "description": "An audio command in an Opus to use for Screencrash",
+    "type": "object",
+    "oneOf": [
+        {
+            "title": "Sync asset",
+            "description": "Sync asset to audio component resource folder. Don't do this manually.",
+            "properties": {
+                "target": { "enum": ["audio"] },
```

Comment:
These enums will always have only one value, right? Then I think it's clearer to use the "const" keyword instead. Same in the other places like this.